docs/running-all-the-test-modules/README.md

```diff
@@ -0,0 +1,121 @@
+# How to run all the test modules?
+
+This guide explains the instructions on running all the test modules without starting the backend and services manually. 
+
+Let's focus this with an example. 
+
+Assume we have ballerina healthcare service backend in ../healthcare/ folder. 
+
+    -> ballerina_integrator
+        -> examples
+            -> guides
+                -> services
+                  -> healthcare-service
+
+#### Requirement: 
+We have set of integration tutorials in example-integration-tutorials folder which includes set of services(bal files) along with their tests. We need to start the services automatically while running the tests including the backend described above. 
+
+#### Why do we need this?
+We have different tutorials as below. Sometimes the same backend resource function can be invoked through different ballerina services. If the backend is running continuously, the data can be duplicated and it can result in giving incorrect test outputs. 
+
+Therefore, through this example we will guide to start the backend service along with the ballerina services in the beginning of the test module. Then backend and the ballerina services will stop at the end of the test module and restarted in the next test module. In this approach the data will not be duplicated since the backend gets restarted at the end of the module. 
```

Comment:
```suggestion
Therefore, through this example, we will guide you to start the backend service along with the ballerina services at the beginning of the test module. Then backend and the ballerina services will stop at the end of the test module and restart in the next test module. In this approach the data will not be duplicated since the backend gets restarted at the end of the module. 
```